Website/Packages/datax-pipeline/src/modules/flowDefinition/api.js

```diff
@@ -11,7 +11,7 @@ import * as Helpers from './flowHelpers';
 // Flow Service
 export const getFlow = name => getProduct(name).then(f => f.gui);
 
-export const saveFlow = config =>
+export const deployFlow = config =>
```

Comment:
Is this a bug by renaming method to deployFlow? Where is saveFlow going? I ask because below, we call the flow/save service API. Why is it not calling something like flow/deploy?

Reply:
renamed to saveFlow